README.md

```diff
@@ -63,14 +63,15 @@ Add [dependency](https://search.maven.org/search?q=a:operator-framework) to your
 </dependency>
 ```
 
-Main method initializing the Operator and registering a controller..
+Main method initializing the Operator and registering a controller.
 
 ```java
 public class Runner {
 
    public static void main(String[] args) {
-       Operator operator = new Operator(new DefaultKubernetesClient());
-       operator.registerController(new WebServerController());
+       Operator operator = new Operator(new DefaultKubernetesClient(),
+           DefaultConfigurationService.instance());
```

Comment:
It's not really easy to understand what DefaultConfigurationService is :(. 
I think there is a way to make better before the final release.

Reply:
The problem is to be able to provide the configuration from the annotation processor in the absence of injection. What would you propose to make this simpler/clearer?

Reply:
`ConfigurationProvider` might be a better name?

Reply:
From the user's point of view, there is only one ConfigurationService in the classpath belonging to the imported dependency, so would be best to hide it completely from the user by default. 
I'm fine with the current approach and naming though, can become simpler in next iterations.

Reply:
@psycho-ir the problem is that we cannot put it there by default because then it would be pulled in by the Quarkus implementation as well, which is exactly what I'm trying to avoid 😄